core/src/SYCL/Kokkos_SYCL.cpp

```diff
@@ -114,10 +114,10 @@ void SYCL::print_configuration(std::ostream& os, bool verbose) const {
 #else
   os << "macro  KOKKOS_IMPL_SYCL_USE_IN_ORDER_QUEUES : undefined\n";
 #endif
-#ifdef SYCL_EXT_ONEAPI_GRAPH
```

Comment:
Don't you care to see whether that macro is defined regardless, as information about the underlying toolchain?
I do realize you could in principle deduce its values from the configuration that is printed, I am asking if you don't deem it useful to have it directly.

Reply:
I think `KOKKOS_IMPL_SYCL_GRAPH_SUPPORT` is more useful enough to report on its own. `SYCL_EXT_ONEAPI_GRAPH` isn't used on its own anymore in `Kokkos`.

Reply:
> `SYCL_EXT_ONEAPI_GRAPH` isn't used on its own anymore in `Kokkos`.

Well you do use it in `Kokkos_Setup_SYCL.hpp` when you decide whether to define `KOKKOS_IMPL_SYCL_GRAPH_SUPPORT`.

Anyway I am not blocking on this.